lib/internal/url.js

```diff
@@ -773,6 +773,7 @@ class URL {
   #context = new URLContext();
   #searchParams;
   #searchParamsModified;
+  static #kParseURLSymbol = Symbol('kParseURL');
```

Comment:
Can you define this outside of the class?

Reply:
resolved